lib/speed/create-results.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const createResult = (
+  // Function which logs results to console
+  results, // Map, results of all functions of all node versions
+  caption, // String
+  count // Number
+) => {
+  const rpad = (s, count) => (s + '.'.repeat(count - s.length));
```

Comment:
Use `padEnd` and `padStart`

Reply:
padEnd and padStart are not implemented in the nodejs 6(version is in the Travis config).